src/components/cl/hier/allreduce/allreduce.h

```diff
@@ -8,8 +8,36 @@
 #define ALLREDUCE_H_
 #include "../cl_hier.h"
 
+enum
+{
+    UCC_CL_HIER_ALLREDUCE_ALG_RAB,
+    UCC_CL_HIER_ALLREDUCE_ALG_SPLIT_RAIL,
+    UCC_CL_HIER_ALLREDUCE_ALG_LAST,
+};
+
+extern ucc_base_coll_alg_info_t
+    ucc_cl_hier_allreduce_algs[UCC_CL_HIER_ALLREDUCE_ALG_LAST + 1];
+
+#define UCC_CL_HIER_ALLREDUCE_DEFAULT_ALG_SELECT_STR "allreduce:0-4k:@rab"
+
 ucc_status_t ucc_cl_hier_allreduce_rab_init(ucc_base_coll_args_t *coll_args,
                                             ucc_base_team_t      *team,
                                             ucc_coll_task_t     **task);
 
+ucc_status_t
+ucc_cl_hier_allreduce_split_rail_init(ucc_base_coll_args_t *coll_args,
+                                      ucc_base_team_t *     team,
+                                      ucc_coll_task_t **    task);
+
```

Comment:
The formating of pointers is not consistent.

Reply:
unfortunately, this is what our clang file produces. i'll fix manually